x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_histogram_panel/index.tsx

```diff
@@ -60,25 +61,39 @@ const ViewAlertsFlexItem = styled(EuiFlexItem)`
   margin-left: ${({ theme }) => theme.eui.euiSizeL};
 `;
 
+const OptionsFlexItem = styled(EuiFlexItem)`
+  margin-left: ${({ theme }) => theme.eui.euiSizeS};
+`;
```

Comment:
It would be so cool if EUI components would come with `margin(Left|Top|Right|Bottom)="s"` props